modules/post/windows/gather/kerberoast.rb

```diff
@@ -0,0 +1,52 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::Powershell
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Common
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name' => 'Kerberoast',
+      'Description' => %q(This module load in memory PowerView.ps1 and executes Invoke-Kerberoast.),
+      'License' => MSF_LICENSE,
+      'Author' => [
+        'harmj0y', # Invoke-Kerberoast
+        'phra' # MSF Module
+      ],
+      'References' => [''],
+      'Platform' => [ 'win' ],
+      'Arch' => [ 'x86', 'x64' ],
+      'SessionTypes' => [ 'meterpreter' ]
```

Comment:
Has it been tested on other session types?

Reply:
no